cpp/src/parquet/types.cc

```diff
@@ -259,6 +271,11 @@ std::string ConvertedTypeToString(ConvertedType::type t) {
   }
 }
 
+int32_t GetCompressionCodecDefaultCompressionLevel(Compression::type t) {
+  return arrow::util::GetCompressionCodecDefaultCompressionLevel(
+      TranslateParquetCompressionTypeToArrowCompressionType(t));
+}
```

Comment:
If my suggested change is made this is not needed

Reply:
Removed.